README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img width="65%" src="docs/source/\_static/img/evalai_logo.png" /></p>
+<p align="center"><img width="65%" src="docs/source/_static/img/evalai_logo.png" /></p>
```

Comment:
The image is not center aligned on README. Please fix. 
<img width="998" alt="Screen Shot 2019-04-11 at 10 36 20 AM" src="https://user-images.githubusercontent.com/2945708/55966021-aa3af480-5c45-11e9-8db5-2e1b41e279bf.png">